src/locales/default.json

```
@@ -375,5 +375,23 @@
     "show": "Show",
     "reply": "Reply",
     "load_more": "Load more"
+  },
+  "page": {
+    "title": {
+      "home": "Home",
```

Home should use the title just "Snapshot"
```suggestion
      "home": "Snapshot",
```